### DISCUSSION TITLE
Proxy provider Review

### DISCUSSION THREAD

**Discussion by SkullFace141** [engageub/InternetIncome#35] [General]
https://www.proxyfish.com/ didn't test, very expensive and look fishy
https://iproyal.com/ Some of their sub-nets work good but you have to contact support before buying to check stock, its expensive
https://proxy-seller.com/ Doesn't work, very bad avoid at all costs
https://app.privateproxy.me/ Doesn't work, very bad avoid at all costs
https://billing.rayobyte.com/ only tested data-center proxies, good but expensive
https://app.proxy-cheap.com/ Doesn't work, very bad avoid at all costs

**Comment by engageub** [comment]
Thank you for providing the review.
Could you please rephrase what you meant by "Does not work". Are the proxies not working or the application detects them as proxies.  Did you test for residential or datacenter. 

Did you sign up using the links mentioned in the [wiki](https://github.com/engageub/InternetIncome/wiki/Proxies-and-Hosting-Providers). 
For [proxyfish](https://www.proxyfish.com/billing/aff.php?aff=449), sign up using the [link](https://www.proxyfish.com/billing/aff.php?aff=449) mentioned and use coupon code ML9G4PE0AG to get 40% discount on checkout.

Thank you

**Comment by SkullFace141** [comment]
unfortunately its one time discount not recurring.
The proxies which doesnt work are very slow and restricted and detected as proxies

**Comment by nkciam** [comment]
Hi @engageub and @SkullFace141, thank you for spend your time review proxy quality. Do you have any update of quality in proxy list recently? It’s good if you can share with us about your experience with the proxy site you are using and it’s working good with some app like earnapp and honeygain. Recently, I’m using proxy in webshare but the quality not good. It’s very slow and the error rate is high ~ 30%. Thank a lot and have a good day!

**Reply by engageub** [reply]
Hi @nkciam ,
Its not only about proxy quality, it also depends on the demand in the location. 
You will have to join different discord groups of respective apps and see their earnings based on location.
Then you can decide on purchasing proxy based on location for the respective app.
Different proxy providers have proxies in different locations. You may choose the proxy provider based on the available location. 

https://github.com/engageub/InternetIncome/wiki/Proxies-and-Hosting-Providers

Thank you
